### DISCUSSION TITLE
小白使用心得（求人不如求己）

### DISCUSSION THREAD

**Discussion by iTooth-x** [ALTaleX531/TranslucentFlyouts#88] [Ideas]
首先要感谢作者给我们带来这么好用的工具。
1、安装和配置问题：安装完没有设置页面，在发布页下载TranslucentFlyoutsConfigV1.3.2进行配置。
2、资源管理器卡死问题：困扰我好几个月了，自打安装win11我就用了恢复经典右键的工具（注册表我嫌麻烦）然后用上美化(枫的工具箱&Translucentflyouts)后就开始随机卡死，那我换个右键工具解决，但是出现导航栏右键卡死，于是换联想右键工具解决，又出现“文件”对话框右键卡死，然后换W11ClassicMenu（其实换了很多个右键工具），起初恢复win11原本右键便可解决，再后来win11原本右键也会卡死，这当中不排除美化软件的问题，但是我就是想用，于是我百度，其中有一项【某些上下文菜单扩展程序可能会与 Windows 11 不兼容，导致右键卡死。您可以禁用这些扩展程序来解决问题】引起我的注意，于是我用360右键管理把右键菜单全部删除（没错我就是懒，教程要求注册表里挨个禁用排除），至此卡死问题解决（以后还会不会出现未知）！
3、很遗憾TranslucentFlyoutsV2和枫的工具箱V2.06b在我电脑上不那么兼容，单独用都很好，一起用就会不起效或部分起效，我只能用TranslucentFlyoutsV1。
![微信截图_20240102145613](https://github.com/ALTaleX531/TranslucentFlyouts/assets/57354277/a33995bc-c28d-4e9e-9541-e5bbccc06a8e)

**Comment by ALTaleX531** [comment]
可以试试V3，我兼容了工具箱
